archinstall/locales/pt/LC_MESSAGES/base.po

```diff
@@ -1198,42 +1198,38 @@ msgstr " - Desativar/Predefinido : 0 ( Desativa as transferências paralelas, pe
 msgid "Invalid input! Try again with a valid input [or 0 to disable]"
 msgstr "Entrada inválida! Tente novamente com uma entrada válida [ou 0 para desativar]"
 
-#, fuzzy
 msgid "Hyprland needs access to your seat (collection of hardware devices i.e. keyboard, mouse, etc)"
-msgstr "O Sway precisa de acesso ao seu \"seat\" (conjunto de dispositivos de hardware, como o teclado, o rato, etc)"
+msgstr "O Hyprland precisa de acesso ao seu \"seat\" (conjunto de dispositivos de hardware, como o teclado, o rato, etc)"
 
-#, fuzzy
 msgid ""
 "\n"
 "\n"
 "Choose an option to give Hyprland access to your hardware"
 msgstr ""
 "\n"
 "\n"
-"Selecionar uma opção para permitir o acesso do Sway ao seu hardware"
+"Selecionar uma opção para permitir o acesso do Hyprland ao seu hardware"
 
-#, fuzzy
 msgid "All entered values can be suffixed with a unit: %, B, KB, KiB, MB, MiB..."
-msgstr "Todos os valores inseridos podem ser sufixados com uma unidade: B, KB, KiB, MB, MiB..."
+msgstr "Todos os valores inseridos podem ser sufixados com uma unidade: %, B, KB, KiB, MB, MiB..."
 
-#, fuzzy
 msgid "Would you like to use unified kernel images?"
-msgstr "Gostaria de usar a swap em zram?"
+msgstr "Gostaria de usar imagens unificadas do kernel?"
 
 msgid "Unified kernel images"
-msgstr ""
+msgstr "Imagens de kernel unificadas"
```

Comment:
Just a small detail, but could be better if "unified kernel images" was translated the same way in both phrases.